docs/services/frontend.md

```diff
@@ -0,0 +1,174 @@
+# frontend
+
+The frontend is responsible to provide a UI for users, as well
+as an API leveraged by the UI or other clients. The application is based on
```

Comment:
After the first sentence, there could be a short, high-level description of the two parts of the instrumentation: browser side and the backend.